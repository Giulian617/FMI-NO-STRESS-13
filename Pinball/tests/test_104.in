100 17081
1 98 8
86 1 7
1 94 8
1 52 5
40 1 5
1 8 8
87 1 3
1 44 10
1 93 5
46 1 8
56 1 2
1 26 7
2 1 1
1 6 4
10 1 5
73 1 1
17 1 9
1 85 10
1 12 4
65 1 10
1 23 3
1 72 1
36 1 9
49 1 1
1 34 3
38 1 1
90 1 10
1 18 4
84 1 7
1 19 8
62 1 4
1 27 6
21 1 2
1 50 9
1 42 5
71 1 2
1 61 10
1 97 3
1 99 7
1 88 7
1 51 7
100 1 4
1 64 5
1 4 6
37 1 2
55 1 10
59 1 1
1 24 1
1 5 9
1 75 4
13 1 9
45 1 9
11 1 5
1 16 4
1 7 2
78 1 8
80 1 5
70 1 8
1 57 10
91 1 3
33 1 2
1 54 8
1 29 5
68 1 4
83 1 1
1 31 3
1 30 7
20 1 3
1 41 6
1 82 9
22 1 5
25 1 2
14 1 8
53 1 3
1 89 5
1 96 1
1 47 7
1 63 2
81 1 10
28 1 6
67 1 7
1 3 4
1 76 1
1 92 2
1 43 8
79 1 9
1 58 9
1 39 7
77 1 4
66 1 9
1 69 4
1 60 1
1 74 3
1 32 2
1 35 3
9 1 10
48 1 6
1 15 2
95 1 8
