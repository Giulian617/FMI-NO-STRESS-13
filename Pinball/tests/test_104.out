992229147
220381021
524922964
856001123
220381021
364854174
47414349
247986594
210191804
210191804
220381021
856001123
47414349
364854174
220381021
856001123
220381021
47414349
664200538
364854174
210191804
664200538
992229147
364854174
975571413
524922964
524922964
210191804
975571413
664200538
364854174
364854174
664200538
664200538
856001123
364854174
992229147
975571413
210191804
524922964
210191804
47414349
247986594
856001123
47414349
975571413
524922964
992229147
856001123
975571413
210191804
664200538
47414349
247986594
364854174
247986594
856001123
992229147
992229147
247986594
220381021
364854174
210191804
247986594
856001123
975571413
220381021
220381021
47414349
364854174
992229147
992229147
664200538
220381021
992229147
220381021
47414349
856001123
210191804
247986594
856001123
992229147
975571413
247986594
975571413
664200538
975571413
210191804
247986594
664200538
364854174
210191804
47414349
47414349
992229147
664200538
47414349
975571413
220381021
